src/Raven.Server/Documents/Handlers/StreamCsvDocumentQueryResultWriter.cs

```diff
@@ -21,12 +23,15 @@ internal class StreamCsvDocumentQueryResultWriter : IStreamDocumentQueryResultWr
         private StreamWriter _writer;
         private CsvWriter _csvWriter;
         private (string, string)[] _properties;
-        private const string FileName = "export.csv";
         private bool writeHeader = true;
-        public StreamCsvDocumentQueryResultWriter(HttpResponse response, Stream stream, DocumentsOperationContext context, string[] properties = null)
+        
+        public StreamCsvDocumentQueryResultWriter(HttpResponse response, Stream stream, DocumentsOperationContext context, string[] properties = null, string csvFileName = "export")
         {
+            csvFileName = $"{csvFileName} {SystemTime.UtcNow.ToString("yyyy-MM-dd HH-mm", CultureInfo.InvariantCulture)}.csv"; 
```

Comment:
let's avoid spaces. Use `_` instead.

Reply:
`Orders_20180522_1213.csv` should be more readable.

Reply:
done